File: Meals_Android V1/app/src/main/java/org/techtown/project5/Network/retrofit/interfaces/Meals.java
package org.techtown.project5.Network.retrofit.interfaces;

import org.techtown.project5.Network.Data;
import org.techtown.project5.Network.response.Response;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface Meals {

    @GET("/meal")
    Call<Response<Data>> GetMeals(@Query("school_id") String school_id,
                                  @Query("office_id") String office_id);
}

File: Meals_Android V2/app/src/main/java/org/techtown/project5/network/retrofit/interfaces/Meals.kt
package org.techtown.project5.network.retrofit.interfaces

import org.techtown.project5.network.Data
import org.techtown.project5.network.Response
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface Meals {

    @GET("v2/meal/yesterday")
    fun getYesterday(@Query("school_id") school_id : String, @Query("office_id") office_id : String): Call<Response<Data>>

    @GET("v2/meal/today")
    fun getToday(@Query("school_id") school_id : String, @Query("office_id") office_id : String): Call<Response<Data>>

    @GET("v2/meal/tomorrow")
    fun getTomorrow(@Query("school_id") school_id : String, @Query("office_id") office_id : String): Call<Response<Data>>
}